lib/tracer.js

```diff
@@ -18,6 +18,7 @@ var options = {
   accountKey: undefined, // Filled in later
   useHttp: false,
   lrtime: lrtime,
+  hostnameOverride: undefined, // Filled in later
```

Comment:
Why fill this in later instead of just putting `process.env.STRONGLOOP_HOSTNAME_OVERRIDE` here?